types/array-flag.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	"strings"
+)
+
+// ArrayFlags can be used when you want to accept array of strings via command line flag
+type ArrayFlags []string
```

Comment:
instead of writing this yourself, I would look at using https://github.com/spf13/viper `StringSlice`

Reply:
- Should we use cobra instead of viper as dynamic buildkite is taking arguments as command line flags and cobra is more suited for that ```https://github.com/spf13/cobra```A Commander for modern Go CLI interactions 

- This would take me sometime to learn this and implement and I would do this in the coming sprint. Please can you confirm your thoughts on this that would be more helpful. Thank You!

Reply:
Hi @sfunkhouser/ @angrieralien  I have added command line flags using cobra also added suggested test cases, Thank you!